spec/graphql/schema/printer_spec.rb

```diff
@@ -63,14 +63,47 @@
       end
     end
 
-    GraphQL::Schema.define(query: query_root, resolve_type: :pass)
+    create_post_mutation = GraphQL::Relay::Mutation.define do
+      name "CreatePost"
+      description "Create a blog post"
+
+      input_field :title, !types.String
+      input_field :body, !types.String
+
+      return_field :post, post_type
+
+      resolve -> (_, _, _) { }
```

Comment:
Could you remove spaces between the arrow and the parenthesis in your lambda definitions please? Thanks 😃